trendyqc/trend_monitoring/backend_utils/plot.py

```
@@ -284,33 +303,19 @@ def format_data_for_plotly_js(plot_data: pd.DataFrame) -> tuple:
-        ],
-        "TruSight Oncology 500": [
-            "#7D8040",   # olive
-            "#00cc99",  # turquoise
-            "#00a600",  # green
-            "#00FF00",   # bright green
+    try:
+        colors = settings.PLOTTING_COLORS
+    except AttributeError:
+        raise ImproperlyConfigured(
+            "PLOTTING_COLORS setting is missing. Please define it in your settings file."
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)